Gruntfile.js

```diff
@@ -60,7 +60,8 @@ module.exports = function(grunt) {
     },
     uglify: {
       options: {
-        banner: '<%= banner %>'
+        banner: '<%= banner %>',
+        screwIE8: false
```

Comment:
I think this has to be under mangle, unless grunt-uglify has that as a top level option.